plotter.py

```diff
@@ -24,17 +26,14 @@ def plot_3D_points(xdata, ydata, zdata):
     plt.show()
 
 '''
-def plot_neutron_flux(3D_matrix_of_flux):
-
-
-plt.imshow(
-x = np.linspace(-1, 1, 100)
-y = np.cos(x)
-
-plt.plot(x, y, 'k.-')
-plt.xlabel('x')
-plt.ylabel('y')
-plt.title('Simple Matplotlib Example')
-plt.show()
+ @brief     Creates a heat map of the flux data
+ @param     flux_data a 3d numpy array containing the flux information
+ @param     index the z cell-value at which the heat map is displayed
 '''
+def plot_heat_map(flux_data, index):
 
+    img = flux_data
+    lum_img = img[:, :, index]
```

Comment:
This works well when I have the mesh spacing set at .1 or higher. If I try to make the spacing lower then .05, I run into a memory error